internal/pkg/api/handleCheckin.go

```diff
@@ -40,7 +40,7 @@ import (
 var (
 	ErrAgentNotFound    = errors.New("agent not found")
 	ErrNoPolicyOutput   = errors.New("output section not found")
-	ErrFailInjectApiKey = errors.New("fail inject api key")
+	ErrFailInjectAPIKey = errors.New("fail inject api key")
```

Comment:
[NIT]
Perhaps `failed to inject api key` or `failure injecting api key`? They feel, to me, more natural to read.